src/js/components/Jokes.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export default class Jokes extends React.Component{
+    render(){
+        let jokes = this.props.jokes
```

Comment:
I found a paragraph about this in the official ES6 (ES2015) specifications : http://www.ecma-international.org/ecma-262/6.0/#sec-automatic-semicolon-insertion , stating that "For convenience, however, such semicolons may be omitted from the source text in certain situations.".

Also, this can be pertinent : https://stackoverflow.com/questions/34950322/use-of-semicolons-in-es6